plugin/external/pluginEntry_test.go

```diff
@@ -181,19 +180,8 @@ func (suite *ExternalPluginEntryTestSuite) TestDecodeExternalPluginEntryWithStat
 	}
 }
 
-func (suite *ExternalPluginEntryTestSuite) TestDecodeExternalPluginEntryWithCacheTTLs() {
```

Comment:
Restore this test.

Reply:
Have thoughts on how I should test that ttl and isInaccessible?

Reply:
Exposing getters for those seem fine.

Reply:
Yeah, getters are fine. I'd add them to methodWrappers though instead of EntryBase.

Reply:
I don't think they're useful generally, they seem fine as part of EntryBase.

Reply:
Was thinking that'd be a nice way to avoid a "plugin authors ignore this stuff" comment, but I think adding that is enough too.

Reply:
Not sure we need to tell people to ignore it. It's pretty basic access, if someone wants to use it they can. I just don't have a reason to make it available more generally.